pgjdbc/src/main/java/org/postgresql/ds/common/BaseDataSource.java

```diff
@@ -1173,13 +1230,17 @@ public void setProperty(PGProperty property, String value) {
     }
     switch (property) {
       case PG_HOST:
-        serverName = value;
+        serverNames = value.split(",");
```

Comment:
This should delegate to the `setServerNames(...)` rather than direct assignment to the member variable.